Serilog.AspNetCore v8.0.0 but assembly version 0.0.0.0

Serilog.AspNetCore v8.0.0 but assembly version 0.0.0.0

**Description**
Got warning during build:
```
C:\Program Files\dotnet\sdk\8.0.101\Microsoft.Common.CurrentVersion.targets(2382,5): warning MSB3277: Found conflicts between different versions of "Serilog.AspNetCore" that could not be resolved. [C:\....\samples\SampleWorkerService\SampleWorkerService.csproj::TargetFramework=net6.0]
C:\Program Files\dotnet\sdk\8.0.101\Microsoft.Common.CurrentVersion.targets(2382,5): warning MSB3277: There was a conflict between "Serilog.AspNetCore, Version=0.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10" and "Serilog.AspNetCore, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10". [C:\W
```

**Reproduction**
dotnet build

**Expected behavior**
Assembly version should corresponding to package version

**Relevant package, tooling and runtime versions**
dotnet SDK 8.0.101

**Additional context**
Inspecting the assembly version

![image](https://github.com/serilog/serilog-aspnetcore/assets/9668549/c959514a-c1ce-4575-816e-1af21f094116)

--- a/src/Serilog.AspNetCore/Serilog.AspNetCore.csproj
+++ b/src/Serilog.AspNetCore/Serilog.AspNetCore.csproj
@@ -13,7 +13,6 @@
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <RepositoryUrl>https://github.com/serilog/serilog-aspnetcore</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <RootNamespace>Serilog</RootNamespace>
     <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>

